autosklearn/pipeline/regression.py

```diff
@@ -262,7 +262,7 @@ def _get_pipeline(self, init_params=None):
 
         steps.extend(
             [["categorical_encoding", OHEChoice(default_dataset_properties)],
-             ["imputation", Imputation()],
+             ["imputation", NumericalImputation()],
```

Comment:
This needs to be adapted to deal with both numerical and categorical features.

Reply:
ok